requirements/static/ci/py3.10/darwin.txt

```diff
@@ -444,7 +444,7 @@ idna==2.8
     #   etcd3-py
     #   requests
     #   yarl
-immutables==0.15
+immutables==0.19
```

Comment:
```
set -l pkg immutables; set -l pkgv 0.15; sed -i "s/$pkg==\(.*\)/$pkg==$pkgv/" requirements/static/*/py3.*/*.txt
```
That's for fish shell, adjust for bash

Reply:
But I don't use fish :)  it stinks
Will go revisit these setting and make them the same as the base 3006.x, thought it would do that with the removal of the line in common.in

Reply:
Done